Source/Cake.CakeMail/Email/CakeMailEmailProvider.cs

```diff
@@ -0,0 +1,66 @@
+﻿using System;
```

Comment:
It's exactly same layering as Cake.SendGrid which, by the way, was modeled after what you did in Cake.Gitter. I personally don't think like this second layer, but when I saw it in your Gitter addin I figured it must be the preferred way so I decided to mimic it. I would prefer to remove this layer from both CakeSendGrid and Cake.CakeMail, let me know what you prefer.